react/components/Table/Toolbar.js

```diff
@@ -370,6 +370,11 @@ Toolbar.propTypes = {
       alignMenu: PropTypes.oneOf(['right', 'left']),
       isLoading: PropTypes.bool,
     }),
+    newLine: PropTypes.shape({
+      label: PropTypes.string,
+      handleCallback: PropTypes.func,
```

Comment:
Same here.

Reply:
this ones can't be required since newLine button is optional. The use of this props is what makes the button appear.